model/OrganisationSearchRequest.ts

```diff
@@ -0,0 +1,23 @@
+import { isly } from "isly"
+import { ProviderCode } from "./ProviderCode"
+
+export interface OrganisationSearchRequest {
+	codes?: string[]
+	activeProviders?: ProviderCode[]
+	statuses?: ("active" | "inactive" | "deleted")[]
```

Comment:
`OrganisationStatusV2` according to mpay-server

Reply:
OrganisationStatus is already changed to lowercase